dynamic-plugins.default.yaml

```diff
@@ -479,6 +479,49 @@ plugins:
               accessKeyId: ${AWS_ACCESS_KEY_ID}
               secretAccessKey: ${AWS_SECRET_ACCESS_KEY}
 
+  - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-cloud
+    disabled: true
+    pluginConfig:
+      enabled:
+        bitbucketCloud: true
+      # Uncomment the following to enable the Bitbucket Cloud catalog
+      # # https://backstage.io/docs/integrations/bitbucketCloud/discovery#configuration
+      # catalog:
+      #   providers:
+      #     bitbucketCloud:
+      #       yourProviderId: # identifies your ingested dataset
+      #         catalogPath: /catalog-info.yaml # default value
+      #         filters: # optional
+      #           projectKey: '^apis-.*$' # optional; RegExp
+      #           repoSlug: '^service-.*$' # optional; RegExp
+      #         schedule: # optional; same options as in TaskScheduleDefinition
+      #           # supports cron, ISO duration, "human duration" as used in code
+      #           frequency: { minutes: 30 }
+      #           # supports ISO duration, "human duration" as used in code
+      #           timeout: { minutes: 3 }
+      #         workspace: workspace-name
+
+  - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-server
+    disabled: true
+    pluginConfig:
+      enabled:
+        bitbucketCloud: true
```

Comment:
shouldn't this be `bitbucketServer` or something else then `bitbucketCloud`?

Reply:
fixed, before we GA we will need to remove the `enabled.xxx` keys anyways.